play/PlayerRules/Basic Rules.txt

```diff
@@ -521,6 +565,51 @@ Hunkering costs one action, and you cannot fire your weapons. Imagine
 curling up in a ball, to pose the absolute smallest target possible. Its a 
 free action to un-hunker
 
+Stealth Attacks:
+If you are stealthed (undetected by an opponent) when you initiate an attack
+action, you incur bonuses depending on the circumstances of the attack.
+	Ranged weapon stealth-attacks (including explosive but not any other thrown 
+weapons) have normal damage with -1 miss chance, if 'adequate' aiming time is 
+given (DM's discretion: how rushed you are, or how skilled you are at stealth, etc).
+	If you have 10 skill in melee combat, Thrown and Melee (non-explosive) weapons
+deal 2x damage, up to 150 maximum damage (before critical effects). If you critically
+hit, or hit at greater than 3 over your miss chance, target gets hit in whatever you
+deem to be the most vulnerable; i.e., the head, neck, or the chest (over the heart,
+etc). If you critical hit, you also gain all the effects of a critical.
+
+Suppression: 
+Every player with a ranged weapon gets the option to Suppress as the last 
+action of their turn. Suppression, also known as, "laying down fire", 
+"covering fire", or "suppressive fire" is the art of shooting a hail of 
+bullets (as accurately as possible) to try and prevent the enemy from 
+effectively fighting back while your team maneuvers. Suppression comes in
+three flavors: light, medium, and heavy; these indicate how wide of an area

```

Comment:
I was going to put the light-medium-heavy suppression definitions in the weapons manual, but I guess it makes sense to define them here